circuitbreakerpanel/circuitbreakerpanel-java-mvn/README.md

```diff
@@ -0,0 +1,41 @@
+# CircuitBreakerPanel recipe for Lagom with Java
+
+CircuitBreakerPanel can be used for with any arbitrary api to apply the circuit breaker pattern to it.
+Instead of wasting resources on a particular part of the service that's failing you can use the circuit breaker pattern to
+fail immediately in case a certain dependency of 1 or more of your endpoints is down.
+
+## Implementation details
+The [`circuitbreakerpanel-impl/src/main/resources/application.conf`](circuitbreakerpanel-impl/src/main/resources/application.conf) contains configuration for your circuitbreaker:
+
+This configuration is specified in detail in the [`lagom documentation`](https://www.lagomframework.com/documentation/1.4.x/java/ServiceClients.html#Circuit-Breaker-Configuration)
+
+In this sample we use the circuitBreakerPanel in [`DemoServceimpl`](circuitbreakerpanel-impl/src/main/java/com/lightbend/lagom/recipes/cbpanel/impl/DemoServiceImpl.java)
+when invoking our repository layer.
+In a real scenario this repository will be interacting with a persistence layer.
```

Comment:
Before this line there are small inconsistencies:

 - `configuration for your circuitbreaker:` - sentence ends on `:`. Should that be a `.` or did you forget a snippet?
 - `circuitBreakerPanel` vs `CircuitBreakerPanel` (capitalization) 
 - CircuitBreakerPanel is a class name so we should format it with backticks: `CircuitBreakerPanel`

I think `CircuitBreakerPanel` adds a new abstraction over the well-known circuit breaker pattern in that it's a registry of circuit breaker's which can be easily reused and accessed. We could mention something in these lines to state how `CircuitBreakerPanel` is something different from a single circuit breaker.

Reply:
Fixed! :smile: